package.json

```diff
@@ -1,15 +1,22 @@
 {
   "name": "amcharts3-angular2",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "description": "Official Angular 2 plugin for amCharts V3",
+  "module": "index.js",
+  "typings": "index.d.ts",
   "repository": {
     "type": "git",
     "url": "https://github.com/amcharts/amcharts3-angular2.git"
   },
-  "peerDependencies": {
-    "@angular/core": "^2.0.0 || ^4.0.0"
```

Comment:
Hmm, and if we have it in `devDependencie`, it's not enough?

Wasn't sure about this one.